### PR TITLE
Added some more API commands

### PR DESCRIPTION
**Pull request by AirTechWick**

### CODE REVIEW

File: fake-theta/commands/listFiles.js [hunk 1]
```diff
@@ -1 +1,15 @@
...
+    }
+}
+const response = await fetch('https://fake-theta.vercel.app/osc/commands/execute', 
+	{method: 'POST', 
+	body: JSON.stringify(body),
+	headers: {'Content-Type': 'application/json'}
+});
+const data =  await response.json();
+
+console.log(data);
```
Comment (codetricity) [inline fake-theta/commands/listFiles.js:15]: use stringify to print out the contents of a JavaScript object to a json string

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify

```javascript
console.log(JSON.stringify(data, null, 4))
```

Refer to the last line of this example:

https://github.com/theta360developers/theta-javascript/blob/main/fake-theta/examples/lastThumbUrl.js

**Review by codetricity — CHANGES_REQUESTED** [review]
output of listFiles should be a readable string.  Use stringify to convert object to string in listFiles.js

**Comment by AirTechWick** [comment]
Fixed requested changes

**Review by jcasman — APPROVED** [review]
Well done! Great use of `stringify`.

**Review by codetricity — APPROVED** [review]
good.  FYI, the number 4 is the number of spaces for the indent of the formatted JSON string output.  If you just use `JSON.stringify(data)`, it will still work, but will not be formatted.  Just in case you want to practice with stringify.

**Comment by codetricity** [comment]
I merged it because @jcasman was in a meeting and it looks like he reviewed it immediately before me.  in general, jesse should merge the pull request after review.

**Comment by jcasman** [comment]
Thank you!

**Pull request merged**